AdNetworkSupport/MoPubMediationLogger.m

```diff
@@ -0,0 +1,124 @@
+//
+// Created by Pooja Shashidhar on 4/16/18.
+// Copyright (c) 2018 MoPub. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "MoPubMediationLogger.h"
+#import "MPLogging.h"
+
+
+@interface MoPubMediationLogger ()
+@property(nonatomic) NSString *NETWORK_TYPE;
+@property(nonatomic) NSString *ADFORMAT_TYPE;
+@property (nonatomic)Event EVENT_TYPE;
```

Comment:
Remove the extra space before `(nonatomic)`.

Reply:
updated.